src/code/z_player_lib.c

```diff
@@ -1,153 +1,1046 @@
+    }
+
+    player->heldItemActionParam = actionParam;
+}
+
+void func_80123DA4(Player* player) {
+    player->unk_730 = NULL;
+    player->stateFlags2 &= ~PLAYER_STATE2_2000;
+}
+
+void func_80123DC0(Player* player) {
+    if ((player->actor.bgCheckFlags & 1) ||
+        (player->stateFlags1 & (PLAYER_STATE1_8000000 | PLAYER_STATE1_800000 | PLAYER_STATE1_200000)) ||
+        (!(player->stateFlags1 & (PLAYER_STATE1_80000 | PLAYER_STATE1_40000)) &&
+         ((player->actor.world.pos.y - player->actor.floorHeight) < 100.0f))) {
+        player->stateFlags1 &= ~(PLAYER_STATE1_40000000 | PLAYER_STATE1_80000 | PLAYER_STATE1_40000 |
+                                 PLAYER_STATE1_20000 | PLAYER_STATE1_10000 | PLAYER_STATE1_8000);
+    } else if (!(player->stateFlags1 & (PLAYER_STATE1_200000 | PLAYER_STATE1_80000 | PLAYER_STATE1_40000))) {
+        player->stateFlags1 |= PLAYER_STATE1_80000;
+    } else if ((player->stateFlags1 & PLAYER_STATE1_40000) && (player->transformation == PLAYER_FORM_DEKU)) {
+        player->stateFlags1 &=
+            ~(PLAYER_STATE1_40000000 | PLAYER_STATE1_20000 | PLAYER_STATE1_10000 | PLAYER_STATE1_8000);
+    }
+
+    func_80123DA4(player);
+}
+
+void func_80123E90(GlobalContext* globalCtx, Actor* actor) {
+    Player* player = GET_PLAYER(globalCtx);
+
+    func_80123DC0(player);
+    player->unk_730 = actor;
+    player->unk_A78 = actor;
+    player->stateFlags1 |= PLAYER_STATE1_10000;
+    func_800DFD78(Play_GetCamera(globalCtx, CAM_ID_MAIN), 8, actor);
+    Camera_ChangeMode(Play_GetCamera(globalCtx, CAM_ID_MAIN), 9);
+}
+
+s32 func_80123F14(GlobalContext* globalCtx) {
+    Player* player = GET_PLAYER(globalCtx);
+
+    return player->stateFlags1 & PLAYER_STATE1_800000;
+}
+
+s32 func_80123F2C(GlobalContext* globalCtx, s32 ammo) {
+    globalCtx->unk_1887C = ammo + 1;
+
+    return 1;
+}
+
+s32 Player_IsBurningStickInRange(GlobalContext* globalCtx, Vec3f* pos, f32 xzRange, f32 yRange) {
+    Player* this = GET_PLAYER(globalCtx);
+    Vec3f diff;
+    s32 pad;
+
+    if ((this->itemActionParam == PLAYER_AP_STICK) && (this->unk_B28 != 0)) {
+        Math_Vec3f_Diff(&this->meleeWeaponInfo[0].tip, pos, &diff);
+        return (SQXZ(diff) <= SQ(xzRange)) && (0.0f <= diff.y) && (diff.y <= yRange);
+    }
+
+    return false;
+}
+
+u8 Player_GetStrength(void) {
+    return sPlayerStrengths[(void)0, gSaveContext.save.playerForm];
+}
+
+u8 Player_GetMask(GlobalContext* globalCtx) {
+    Player* player = GET_PLAYER(globalCtx);
+
+    return player->currentMask;
+}
+
+// Unused
+void Player_RemoveMask(GlobalContext* globalCtx) {
+    Player* player = GET_PLAYER(globalCtx);
+
+    player->currentMask = PLAYER_MASK_NONE;
+}
+
+s32 Player_HasMirrorShieldEquipped(GlobalContext* globalCtx) {
+    Player* player = GET_PLAYER(globalCtx);
+
+    return (player->transformation == PLAYER_FORM_HUMAN) && (player->currentShield == PLAYER_SHIELD_MIRROR_SHIELD);
+}
+
+s32 Player_HasMirrorShieldSetToDraw(GlobalContext* globalCtx) {
```

Comment:
I think `Player_IsHoldingMirrorShield` would be a better name, see discussion here:
https://discord.com/channels/688807550715560050/688851317593997489/974840246657232967

Same with:
`Player_HoldsHookshot` -> `Player_IsHoldingHookshot`
`Player_HoldsTwoHandedWeapon` -> `Player_IsHoldingTwoHandedWeapon`